csvSplit.py

```diff
@@ -96,7 +96,18 @@ def _split_fileSplits_withPrefix(game: str, seg: str, categoryPrefix: str):
                 f.write(f"offset,vram,.{section}\n")
                 for row in data:
                     offset, vram, filename = row
-                    f.writelines(f"{offset:X},{vram:X},{filename}\n")
+                    # z_message_PAL.data is included with .rodata in the
+                    # spreadsheet, because that's where it's located in the ROM
+                    if filename == 'z_message_PAL.data':
+                        f.write("\n")
+                        f.writelines(f"offset,vram,.data\n")
+                        f.writelines(f"{offset:X},{vram:X},z_message_PAL\n")
+                    elif filename == 'z_message_PAL.rodata':
+                        f.write("\n")
+                        f.writelines(f"offset,vram,.rodata\n")
+                        f.writelines(f"{offset:X},{vram:X},z_message_PAL\n")
+                    else:
+                        f.writelines(f"{offset:X},{vram:X},{filename}\n")
```

Comment:
Could you add a check for the file being `oot` too, I'm worried this may mess up mm csv parsing